Cargo.toml

```diff
@@ -49,6 +50,9 @@ serde_derive = "1.0.0"
 serde = "1.0.0"
 clippy = { version = "=0.0.142", optional = true }
 chrono = "0.4.0"
+pulldown-cmark = { version = "0.0.15", default-features = false }
+ammonia = "0.5.0"
+syntect = "1.7.1"
```

Comment:
I don't know, I just copied and pasted the requirement line from the crates' pages ([example](https://crates.io/crates/ammonia "Ammonia's crate page")). If I understand correctly the [cargo documentation](http://doc.crates.io/specifying-dependencies.html#specifying-dependencies-from-cratesio "Specifying dependencies from crates.io"), this is just a stylistic issue. I'd be happy to change it if that's what we want, but I'm not sure this is needed.